torch/_dynamo/variables/builtin.py

```diff
@@ -10,10 +10,13 @@
 import types
 from collections import defaultdict, OrderedDict
```

Comment:
This still has `# mypy: ignore-errors` at the top of the file - intended ?

Reply:
Yes because I only annotate tx on this PR, there's A TON more work to be done to clean these files